app/cafe.py

```diff
@@ -0,0 +1,32 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    NotWearingMaskError,
+    OutdatedVaccineError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(
+                f"Vaccine is required to visit {self.name}"
+            )
+
+        if (visitor.get("vaccine").get("expiration_date")

```

Comment:
In case you aim to make a more error-tolerant approach using `get`, add a default `dict` that is returned in case `vaccine` key is absent. It will allow the second `.get()` to not fail. 
```suggestion
        if (visitor.get("vaccine", {}).get("expiration_date")

```